### PR TITLE
Create test.txt

### PR DESCRIPTION
**Pull request by alita-moore**
When opening a pull request to submit a new EIP, please use the suggested template: https://github.com/ethereum/EIPs/blob/master/eip-template.md

We have a GitHub bot that automatically merges some PRs. It will merge yours immediately if certain criteria are met:

 - The PR edits only existing draft PRs.
 - The build passes.
 - Your GitHub username or email address is listed in the 'author' header of all affected PRs, inside <triangular brackets>.
 - If matching on email address, the email address is the one publicly listed on your GitHub profile.

### CODE REVIEW

**Comment by alita-moore** [comment]
testsetset

**Review by alita-moore — COMMENTED** [review]
this is a review / comment

**Review by fake-alita-moore — APPROVED** [review]
you are best programminer

**Review by fake-alita-moore — CHANGES_REQUESTED** [review]
nvrmnd boooo

**Review by fake-alita-moore — COMMENTED** [review]
dis is comment

File: EIPS/eip-6969.md [hunk 1]
```diff
@@ -0,0 +1,316 @@
...
+
+* **Mismatch between volatility of transaction fee levels and social cost of transactions**: bids to include transactions on mature public blockchains, that have enough usage so that blocks are full, tend to be extremely volatile. On Ethereum, minimum bids range between 1 nanoeth (10^9 nanoeth = 1 ETH), but sometimes go over 100 nanoeth and have reached over 200 nanoeth. This clearly creates many inefficiencies, because it's absurd to suggest that the cost incurred by the network from accepting one more transaction into a block actually is 200x more when gas prices are 200 nanoeth than when they are 1 nanoeth; in both cases, it's a difference between 8 million gas and 8.02 million gas.
+* **Needless delays for users**: because of the hard per-block gas limit coupled with natural volatility in transaction volume, transactions often wait for several blocks before getting included, but this is socially unproductive; no one significantly gains from the fact that there is no "slack" mechanism that allows one block to be bigger and the next block to be smaller to meet block-by-block differences in demand.
```
Comment (fake-alita-moore) [inline EIPS/eip-6969.md:28]: testing 1 02 4

Comment (fake-alita-moore) [reply EIPS/eip-6969.md:28]: asdfasdfasdfdf

**Comment by alita-moore** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):


	 - File with name EIPS/eip-6969.md is new and new files must be reviewed
	 - Filename test.txt is not in EIP format 'EIPS/eip-####.md'
	 - eip-1559.md requires approval from one of (vbuterin,econoar,afdudley,mslipper,i-norden,abdelhamidbakhta)

**Comment by alita-moore** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):


	 - File with name EIPS/eip-6969.md is new and new files must be reviewed
	 - Filename test.txt is not in EIP format 'EIPS/eip-####.md'
	 - eip-1559.md requires approval from one of (vbuterin,econoar,afdudley,mslipper,i-norden,abdelhamidbakhta)

**Comment by alita-moore** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):


	 - File with name EIPS/eip-6969.md is new and new files must be reviewed
	 - Filename test.txt is not in EIP format 'EIPS/eip-####.md'
	 - eip-1559.md requires approval from one of (vbuterin,econoar,afdudley,mslipper,i-norden,abdelhamidbakhta)

**Comment by alita-moore** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):


	 - File with name EIPS/eip-6969.md is new and new files must be reviewed
	 - Filename test.txt is not in EIP format 'EIPS/eip-####.md'
	 - eip-1559.md requires approval from one of (vbuterin,econoar,afdudley,mslipper,i-norden,abdelhamidbakhta)

**Comment by alita-moore** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):


	 - File with name EIPS/eip-6969.md is new and new files must be reviewed
	 - Filename test.txt is not in EIP format 'EIPS/eip-####.md'
	 - eip-1559.md requires approval from one of (vbuterin,econoar,afdudley,mslipper,i-norden,abdelhamidbakhta)

**Comment by alita-moore** [comment]
Hi! I'm a bot, and I wanted to automerge your PR, but couldn't because of the following issue(s):


	 - File with name EIPS/eip-6969.md is new and new files must be reviewed
	 - Filename test.txt is not in EIP format 'EIPS/eip-####.md'
	 - eip-1559.md requires approval from one of (vbuterin,econoar,afdudley,mslipper,i-norden,abdelhamidbakhta)

**Pull request merged**